pkg/cmd/grafana-cli/commands/conflict-examples/conflict_example_users.tf

```diff
@@ -15,50 +15,16 @@ provider "grafana" {
 // login conflict
 // Creating the grafana-login
 resource "grafana_user" "grafana-login" {
-  email    = "grafana_login@grafana.com"
-  login    = "GRAFANA_LOGIN@grafana.com"
+  email    = "grafana_email_1@grafana.com"
+  login    = "GRAFANA_LOGIN"
   password = "grafana_login@grafana.com"
   is_admin = false
 }
 
 // Creating the grafana-login
 resource "grafana_user" "grafana-login-2" {
-  email    = "grafana_login_2@grafana.com"
-  login    = "grafana_login@grafana.com"
+  email    = "grafan_email_2@grafana.com"
+  login    = "grafana_login"
   password = "grafana_login@grafana.com"
   is_admin = false
 }
-
```

Comment:
yea, good thinking. I wanted to show all the different cases here and this makes it clear how it can be setup.
Thanks for the comment. I'l revert this and make sure that the login conflicts are clearly about login details and remove any resemblance of an email